fixtures/parser_tests.js

```diff
@@ -372,7 +330,7 @@ function parserTests(parse: (file: string) => ParserResult) {
         22,
         7,
         23,
-        18,
+        18
```

Comment:
Is there any reason for removing trailing commas?

Reply:
i double checked and it seems to be defined in the eslint / prettier config like this. i just jumped into the file and autosave triggered autoFixOnSave which triggered eslint & prettier to fix the linting / formatting according to the defined config.

If i add this trailing comma, vscode tells me that there is a eslint issue.